libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFNetwork.m

```diff
@@ -41,6 +45,41 @@ @implementation SFNetwork
 static NSURLSessionConfiguration *kSFSessionConfig;
 __weak static id<SFNetworkSessionManaging> kSFNetworkManager;
 
++ (instancetype)defaultEphemeralNetwork {
+    NSURLSessionConfiguration *sessionConfiguration = [NSURLSessionConfiguration ephemeralSessionConfiguration];
+    return [[self alloc] initWithSessionIdentifier:kSFNetworkEphemeralSessionIdentifier sessionConfiguration:sessionConfiguration];
+}
+
++ (instancetype)defaultBackgroundNetwork {
```

Comment:
This method should simply call the other new class method (`networkWithSessionIdentifier`), passing in the default identifier and a default session configuration.